src/utils/profileUtils.ts

```diff
@@ -40,3 +40,103 @@ export const userToCollaborator = (
     orgName: user?.organization?.orgName,
   },
 });
+
+/**
+ * The data structure for a columized PBAC group
+ */
+export type ColumnizedPBACGroups<T = unknown> = {
+  /**
+   * The name of the group of PBACDefaults
+   *
+   * All PBACDefaults in the group will have the same group name
+   */
+  name: string;
+  /**
+   * The PBACDefaults for the group
+   */
+  data: PBACDefault<T>[];
+}[][];
+
+/**
+ * A utility function to group an array of PBACDefaults into columns
+ * based on the group name.
+ *
+ * If colCount is greater than the number of groups, the last column will
```

Comment:
I could be misunderstanding, but I believe it should be reversed since you are looping through the groups and checking if that is greater. Ex. "If number of groups is greater than colCount, ...".

Although, your implementation does seem correct and working as expected.

Reply:
You're correct. I wrote the description then didn't proof read it. Let me know if this makes more sense:

>  * If the number of unique groups exceeds `colCount`, the function will
>  * aggregate the remaining groups into the last column.

Updated in 99a161f.

Reply:
LGTM!